Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesVfr.vfr

```diff
@@ -177,4 +183,22 @@ formset
     subtitle text = STRING_TOKEN(STR_EXIT_STRING);
   endform;
 
+  form formid = DASHARO_POWER_CONFIGURATION_FORM_ID,
+       title  = STRING_TOKEN(STR_DASHARO_POWER_CONFIGURATION_TITLE);
+
+    subtitle text = STRING_TOKEN(STR_EMPTY_STRING);
+
+    oneof varid  = FeaturesData.FanCurveOption,
+             prompt = STRING_TOKEN(STR_FAN_CURVE_PROMPT),
+             help   = STRING_TOKEN(STR_FAN_CURVE_HELP),
+             flags  = INTERACTIVE,
+
+            option text = STRING_TOKEN(STR_FAN_CURVE_SILENT), value = FAN_CURVE_OPTION_SILENT, flags = 0;
+            option text = STRING_TOKEN(STR_FAN_CURVE_PERFORMANCE), value = FAN_CURVE_OPTION_PERFORMANCE, flags = 0;

```

Comment:
The indentation here is off a little bit, please fix @mkopec

Reply:
fixed here and in the ME section above